man/vendor/github.com/russross/blackfriday/v2/README.md

```diff
@@ -192,7 +192,7 @@ Extensions
 In addition to the standard markdown syntax, this package
 implements the following extensions:
 
-*   **Intra-word emphasis supression**. The `_` character is
+*   **Intra-word emphasis suppression**. The `_` character is
```

Comment:
This file is coming from a dependency that's vendored, so should not be modified.

Can you refer this part of the change and amend the commit (so that there's only a single commit in the branch)?